docs/advanced/developer-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Developer guide"
+layout: default
+sort: 1
+---
+
+# Developer guide
+
+{: .no_toc }
+
+## Table of contents
+
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+## Building the operator
+
+### Download the source code
+
+```bash
+git clone https://github.com/kubernetes-sigs/node-feature-discovery-operator
+```
+
+### Build the operator image
+
+```bash
+IMAGE_REGISTRY=<my registry>
+make image
+```
+
+#### Push the container image
+
+```bash
+IMAGE_REGISTRY=<my registry>
+make push
+```
+
+Alternatively, instead of specifying variables on the command line,
+you can edit the Makefile to permanently change parameter defaults
+like name of the image or namespace where the operator is deployed.
+
+## Manual deploy of the operator
+
+After building the image you can simply run
+
+```bash
+IMAGE_REGISTRY=<my registry>
+make deploy
+```
+
+Then create a NodeFeatureDiscovery CR by running
+
+```bash
+kubectl apply -f config/samples/nfd.kubernetes.io_v1_nodefeaturediscovery.yaml
+```
+
+## Undeploy de operator
```

Comment:
s/de/the/

Reply:
my Spanish brain won the battle here ;P sorry